source/ceylon/time/Date.ceylon

```diff
@@ -32,6 +32,15 @@ shared interface Date
 
     "Returns the [[DateRange]] between this and given [[Date]]."
     shared formal DateRange rangeTo( Date other );
+    
+    "Two `Date`s are considered equals if they have the same 
```

Comment:
Well, the thing is that using term _day of era_ in the specification of equals says, _"Date equality is based on the equality of `dayOfEra` attribure of the `Date` instance"_.

But that might not be strictly right, because a date might have more than one _Eras_ and a particular value of `dayOfEra` might not be enough to declare two dates equal. 

I.e. _March 12, 102 BC_ is not same as _March 12 102 AD_ (if we ever need to split BC and AD into separate eras).